#l_clk = `left` motor `clockwise`
#l_aclk= 							`anti-clockwise`
#r_clk = `right`

uint16 l_clk
uint16 l_aclk
uint16 r_clk
uint16 r_aclk
